german/the-black-hack.md

```diff
@@ -57,38 +57,38 @@ Monster haben 1 Rüstungspunkt für jeden **TW** über 1, also einfach für den
-
-
-| GEGENSTAND                   | KOSTEN | VORRATSWÜRFEL |         ANMERKUNG          |
-|:-----------------------------|:------:|:-------------:|:--------------------------:|
-| Festes Wams                  |   50   |       -       |      2 Rüstungspunkte      |
-| Lederrüstung                 |  100   |       -       |      4 Rüstungspunkte      |
-| Kettenhemd                   |  350   |       -       |      6 Rüstungspunkte      |
-| Plattenrüstung               |  600   |       -       |      8 Rüstungspunkte      |
-| Kleiner/Großer Schild        | 50/100 |       -       |     2/4 Rüstungspunkte     |
-| Rucksack                     |   5    |       -       |        Traglast +2         |
-| Lampenöl                     |   2    |      d6       |              -             |
-| Handwerksgerät               |   2    |       -       |              -             |
-| Heiliges Symbol              |   25   |       -       |  +2 auf WEI beim Bannen    |
-| Weihwasser                   |   25   |      d4       |              -             |
-| Kletterhaken                 |   1    |       -       |              -             |
-| Laterne                      |   10   |       -       |              -             |
-| Handspiegel                  |   5    |       -       |              -             |
-| Haltbare Rationen            |   15   |      d8       |              -             |
-| Frische Rationen             |   5    |      d4       |              -             |
-| 15m Seil                     |   1    |       -       |              -             |
-| Beutel                       |   1    |       -       |              -             |
-| Sack                         |   2    |       -       |              -             |
-| Diebeswerkzeug               |   25   |       -       |              -             |
-| Feuerstein & Stahl           |   3    |       -       |              -             |
-| Fackeln (6)                  |   1    |      d6       | 1 Vorratswürfel pro Fackel |
-| Wasserschlauch               |   1    |      d6       |              -             |
-| Wein                         |   1    |       -       |              -             |
-| Auswahl gewöhnlicher Kräuter |   10   |      d8       |              -             |
-| 3m Holzstange                |   1    |       -       |              -             |
-| Köcher mit Pfeilen/Bolzen    |   10   |      d10      |              -             |
-| Zweihändige Waffe            |   50   |       -       |              -             |
+Wenn Charaktere Rüstung tragen, die bei ihrer Klasse nicht angegeben ist, addieren sie all ihre Rüstungspunkte (egal wie viele sie davon verbraucht haben) auf alle Würfelergebnisse zum **Angreifen** oder **Verteidigen**.
```

Comment:
Hm. Stimmt, in dem Kontext wäre es Verteidigen, aber bei Schutz vor Bösem ist ganz klar mehr als das gemeint. Um bei einheitlichen Begriffen zu bleiben würde ich das nicht ändern.